uhu/config.py

```diff
@@ -12,7 +12,7 @@
 
 
 class Config:
-    """This is the wrapper to manage ~/.uhu configuration file."""
+    """This is the wrapper to manage ~/.uhu-config configuration file."""
```

Comment:
Instead, we could use `.config/uhu` as this has become the default place to store settings.